src/app/profile/email-verification/email-verification.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  ActivatedRoute,
+  Router
+} from '@angular/router';
+
+@Component({
+  selector: 'fabric8-cleanup',
+  templateUrl: 'email-verification.component.html',
+  styleUrls: ['./email-verification.component.less']
+})
+export class EmailVerificationComponent implements OnInit {
+
+  private verified: string = 'false';
+  private message: string = 'Your e-mail has been confirmed.';
+  private secMessage: string = 'Thank you for validating your e-mail address.' +
+  'You can now continue to use Openshift.io';
+
+  constructor(
+    private route: ActivatedRoute) {}
+
+  ngOnInit(): void {
+    const queryParams = this.route.snapshot.queryParams;
+    if (this.route.snapshot.queryParams['verified']) {
+      this.verified = this.route.snapshot.queryParams['verified'];
+      if (this.verified === 'false') {
+        this.message = this.route.snapshot.queryParams['error'];
+        this.secMessage = 'Your e-mail has already been approved. ' +
```

Comment:
So, this error message is still being hardcoded? UI can't know what is the error. Just render whatever error message is returned by backend. It's not necessarily a problem with already approved email.

Reply:
?

Reply:
@SMahil I agree with @alexeykazakov. Can you please work with @sbose78 and @rohitkrai03 to understand how to get the error message into the UI.

Reply:
@sahil143 , the redirected url only has one error message which you get in 
`this.route.snapshot.queryParams['error']` . 

I think `this.secMessage` should be some generic message for success/failure.